consensus/safety-rules/src/test_utils.rs

```diff
@@ -224,3 +226,12 @@ pub fn test_storage(signer: &ValidatorSigner) -> PersistentSafetyStorage {
         waypoint,
     )
 }
+
+pub fn test_serializer(signer_byte: u8, verify_vote_proposal_signature: bool) -> SerializerService {
+    let signer = ValidatorSigner::from_int(signer_byte);
```

Comment:
I suggest not fuzzing this part and always using the same signer

Reply:
it'll also be faster (if you use once_cell::lazy for that)

Reply:
Sure, thanks -- I was originally thinking of fuzzing these values to increase the coverage of possible Serializer states, but I've just kept it simple for now and created these things with constants (e.g., the signer uses the int 0). We can see if we need to improve this later, once we look at the coverage